openslides/assignments/projector.py

```diff
@@ -27,3 +28,9 @@ def get_requirements(self, config_entry):
             for poll in assignment.polls.all().prefetch_related('options'):
                 for option in poll.options.all():
                     yield option.candidate
+                for speaker in assignment.agenda_item.speakers.filter(end_time=None):
```

Comment:
The ListOfSpeakersOverlay does not know which other projector elements exist, so they cannot easily get `agenda_item.speakers`. As the list of speakers will be dependant on any projector (not always the one with the overlay), I think it is better to wait for multiprojector here. I'll discuss it with @FinnStutzenstein.